### DISCUSSION TITLE
[Exercise-4]: Write a sample PL/SQL block using SELECT INTO and share your examples.

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#5] [General]
Ref.: https://www.youtube.com/watch?v=S0XnhFOmCJ0&list=PLF6_5RI6ENIlDi3knzcQGUiFc5-WdA_1W&index=1&t=2444s

**Comment by srideviadapa0511** [comment]
![SELECT INTO](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/4808266b-ed78-40a8-8b67-07b69c2bc973)

**Comment by cm2024jsp** [comment]
CREATE TABLE employee_data (
    employee_id NUMBER,
    first_name VARCHAR2(50),
    last_name VARCHAR2(50),
    salary NUMBER
);


DECLARE
    v_employee_id NUMBER := 101;
    v_first_name VARCHAR2(50) := 'John';
    v_last_name VARCHAR2(50) := 'Doe';
    v_salary NUMBER := 50000;
BEGIN
    
    INSERT INTO employee_data (employee_id, first_name, last_name, salary)
    VALUES (v_employee_id, v_first_name, v_last_name, v_salary);
    
END;
/

SELECT * FROM employee_data;

**Reply by acrlakshman** [reply]
@cm2024jsp , nice work in demonstrating the use of PL/SQL block.

But, the question also asks about using `INTO` clause. Can you write one demonstrating that?

**Reply by cm2024jsp** [reply]
sir we already have used INTO CLAUSE in the begin block

**Reply by acrlakshman** [reply]
@cm2024jsp Yes. I was referring to `SELECT` `INTO`, for e.g. `SELECT first_name INTO v_first_name FROM employee_data WHERE employee_id=101`.

**Reply by acrlakshman** [reply]
Never mind @cm2024jsp I saw your response here: https://github.com/exaslate-courses/oracle-sql-plsql-b3/discussions/8#discussioncomment-7212152

**Comment by YaminiYadavv** [comment]


**Reply by acrlakshman** [reply]
@YaminiYadavv the example you presented here does not refer to the usage of `SELECT INTO` clause. You must have a PL/SQL code. Please refer to the above link for the usage and reupload this exercise.

**Reply by YaminiYadavv** [reply]
ok sir
